api/api/services/internal_tasks/meta_agent_service.py

```diff
@@ -1412,10 +1550,182 @@ async def stream_proxy_meta_agent_response(
                 yield []
                 return
 
-        else:
-            # This is an actual client message. Answer
-            instructions = GENERIC_INSTRUCTIONS
-            message_kind = "non_specific"
+        else:  # Message is user-triggered
```

Comment:
@guillaq this is pretty verbose but this is what you get when you mix agentic and deterministic flows...